exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
-
-console.log(message);
+let myName="mari"
+let lengthName=myName.length
+// let messageTwo=`my name is${myName}and my name is${lengthName}characters long`
+let message="        My name is"+myName+" and my name is " + lengthName+" characters long"
```

Comment:
This is a very small comment, but you should try to be consistent with your style. If you are putting spaces before and after the plus signs, for example. `is" + myName + " an` you should do that for all strings concatenations.

Reply:
I used the `.trim` method to remove the extra whitespace for this I made space. so I should do that for all strings concatenations??

Reply:
Thanks @mcarballopacheco Your suggestion is quite right. I'm practising consistency.